doc/Makefile

```diff
@@ -3,8 +3,8 @@
 
 # You can set these variables from the command line.
 PYTHON        ?= python
-SPHINXOPTS    ?= -j 1
-SPHINXBUILD   ?= python $(shell which sphinx-build)
+SPHINXOPTS    ?= -j $(shell nproc)
```

Comment:
This doesn't really belong in this PR and seems to have been done in #3478 whats up with github?